trendy-fashion/src/Pages/HomePage.jsx

```diff
@@ -1,377 +0,0 @@
-        ))}
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/060123-D-UHP-topshelf-header.jpg" alt="" />
-      </div>
-
-      <div className='imgdiv4' style={{ display: 'flex' }}>
-        {B17.map(el => (
-          <img style={{ height: '450px' }} key={el.id} alt="" src={el.photo} />
-        ))}
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/D-UHP-classicbrands-header.jpg" alt="" />
-      </div>
-
-      <div className='imgdiv4' style={{ display: 'flex' }}>
-        {B18.map(el => (
-          <img style={{ height: '450px' }} key={el.id} alt="" src={el.photo} />
-        ))}
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/060123-D-UHP-home-header.jpg" alt="" />
-      </div>
-
-      <div className='imgdiv4' style={{ display: 'flex' }}>
-        {B19.map(el => (
-          <img style={{ height: '450px' }} key={el.id} alt="" src={el.photo} />
-        ))}
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/060123-D-UHP-discoveronlinestores-header.jpg" alt="" />
-      </div>
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/060123-D-UHP-discoveronlinestores-AJIOGold.jpg" alt="" />
-      </div>
-
-
-      <div style={{ display: 'grid', gridTemplateColumns: '1fr 1fr 1fr' }}>
-        {B20.map(el => (
-          <img style={{ height: '600px', width: '620px' }} key={el.id} alt="" src={el.photo} />
-        ))}
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/Discover%20Online%20Stores%20(D)111.jpg" alt="" />
-      </div>
-
-      <div>
-        <img src="https://assets.ajio.com/cms/AJIO/WEB/08032022-D-unisex-ajiocares-strip%20new%20d.jpg" alt="" />
-      </div>
-
-      <button className='scrollbtn'
-        onClick={() => {
-          window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
-        }}>▲</button>
-
-      <Popup trigger={<button className='scrollbtn1'>■ ■ ■</button>} position="center center">
-        <div style={{ padding: '60px 160px 60px 160px', marginLeft: '-750px', marginTop: '-500px', marginRight: '150px', backgroundColor: 'white', borderRadius: '20px' }}>
-          <div style={{ margin: 'auto', fontSize: '28px', textAlign: 'center', marginBottom: '70px' }}>
-            <h1>Who do you want to shop for?</h1>
-          </div>
-          <div style={{ display: 'flex' }}>
-            {B21.map(el => (<>
-              <div style={{ textAlign: 'center' }}>
-
-                <NavLink to={el.nav}>
-                  <h1 style={{ fontSize: '23px', marginBottom: '20px' }}>{el.title}</h1>
-                </NavLink>
-                <div style={{ padding: '5px' }}>
-                  <img style={{ height: '250px', width: '220px' }} key={el.id} alt="" src={el.photo} />
-                </div>
-              </div>
-            </>
-            ))}
-          </div></div>
-      </Popup>
-    </>
-  )
```

Comment:
If possible to break this components into multiple smaller components, then please do it